charts/kube-prometheus-stack/README.md

```diff
@@ -80,9 +82,30 @@ _See [helm upgrade](https://helm.sh/docs/helm/helm_upgrade/) for command documen
 
 A major chart version change (like v1.2.3 -> v2.0.0) indicates that there is an incompatible breaking change needing manual actions.
 
+### From 45.x to 46.x
+
+This version upgrades Prometheus-Operator to v0.65.1 with new CRDs (PrometheusAgent and ScrapeConfig), Prometheus to v2.43.1 and Thanos to v0.31.0.
+
+As of this release, CRD installation and upgrade is now managed by the chart if the property `crds.enabled` is set to true (by default).
```

Comment:
Do you also check for the existing helm property `skipCrds` (can you even do that?). If you can't do that you should mention that users of `skipCRD` need to migrate to `.Values.crds.enabled`. Otherwise users that already don't install the CRDs using this chart might suddenly have their CRDs managed by the chart by accident.

Reply:
Now I realize that changelogs need to be read carefully especially when it concerns a breaking change however the breaking change churn on this chart is so large that I can imagine that users might be a bit changelog-fatigued when it comes to this chart.